pantomime-lib/Framework/Pantomime/CWMessage.m

```diff
@@ -1427,6 +1427,9 @@ - (void) addHeadersFromData: (NSData *) theHeaders  record: (CWCacheRecord *) th
             aData = [CWParser parseSubject: aLine  inMessage: self  quick: NO];
             if (theRecord) theRecord.subject = aData;
         }
+        else if ([aLine hasCaseInsensitiveCPrefix: "Content-Type"] && [[aLine asciiString] containsString:@"multipart/signed"] && [[aLine asciiString] containsString:@"application/pkcs7-signature"]) {
```

Comment:
This would match "Content-TypeGARGLE" as well. There are better checks already implemented that correctly parse a header.

The same for using `containsString`.